tools/src/main/java/com/example/core/CompletionStageUtils.java

```diff
@@ -28,11 +28,11 @@ private CompletionStageUtils() {
         };
     }
 
-    public static <T> CompletionStage<Done> doAll(CompletionStage<T>... stages) {
+    public static CompletionStage<Done> doAll(CompletionStage<?>... stages) {
         return doAll(Arrays.asList(stages));
     }
 
-    public static <T> CompletionStage<Done> doAll(List<CompletionStage<T>> stages) {
+    public static CompletionStage<Done> doAll(List<CompletionStage<?>> stages) {
```

Comment:
Again, in this file we were getting warnings because of the combination of varargs and generics. In fact, we don't care about the result types of the completion stages, because these methods throw out the results and replace them with `Done` anyway.